templates/default/stunnel.conf.erb

```diff
@@ -63,6 +63,11 @@ debug = <%= node['stunnel']['debug'] %>
 output = <%= node['stunnel']['output'] %>
 <% end -%>
 
+; Enable/disable FIPS mode
+<% unless node['stunnel']['fips'].nil? %>
```

Comment:
Oh, and one other reason to default ['stunnel']['fips'] to nil leaving it out of the config entirely is that the default setting for fips changes between stunnel versions:

- In 4.x fips is set to "yes" if FIPS is compiled into stunnel.
- In 5.x fips is set to "no".

Making the default "yes" or "no" (instead of leaving it out entirely) would introduce breaking behavior for users of the cookbook. If a user of this cookbook running stunnel 4.x with FIPS compiled in were to use this cookbook and they upgraded, suddenly fips would be turned off.

Perhaps I should add a comment to the code giving just a bit of explanation?